### PR TITLE
docs: add documentation for oras completion

### PR DESCRIPTION
**Pull request by deepeshaburse**
This PR adds documentation for `oras completion`.

### CODE REVIEW

**Comment by deepeshaburse** [comment]
The order of the commands will be correct after the PR #245 gets merged.

**Comment by deepeshaburse** [comment]
cc: @TerryHowe

**Review by TerryHowe — COMMENTED** [review]
Most of the commands pages are pretty much generated from running the help on the command. The format of this doesn't really match the rest of the commands. This reads more like a how-to which I think would be useful.

File: docs/commands/oras_completion.mdx [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+title: oras completion
+sidebar_position: 45
+---
+
+# oras completion
+
+`oras completion` helps to generate the autocompletion script for oras for the specified shell.
```
Comment (TerryHowe) [inline docs/commands/oras_completion.mdx:8]:
```suggestion
`oras completion` helps to generate the autocompletion script for oras for the specified shell and operating system.
```

**Review by TerryHowe — DISMISSED** [review]
/lgtm

File: docs/commands/oras_completion.mdx [hunk 2]
```diff
@@ -0,0 +1,33 @@
+---
+title: oras completion
+sidebar_position: 45
+---
+
+# oras completion
+
+`oras completion` helps to generate the autocompletion script for oras for the specified shell and operating system.
```
Comment (sajayantony) [inline docs/commands/oras_completion.mdx:8]:
```suggestion
`oras completion` helps to generate the autocompletion script for oras for the specified shell.
```

**Review by TerryHowe — APPROVED** [review]
/lgtm

File: docs/commands/oras_completion.mdx (hunk 2 above)
Comment (TerryHowe) [reply docs/commands/oras_completion.mdx]: Okay

**Review by sajayantony — APPROVED** [review]

**Pull request merged**